Library/Homebrew/cmd/update.sh

```diff
@@ -336,7 +336,8 @@ homebrew-update() {
       *)
         odie <<EOS
 This command updates brew itself, and does not take formula names.
-Use 'brew upgrade <formula>'.
+Use 'brew upgrade <formula>' instead.
+You probably want 'brew upgrade $@'...
```

Comment:
`Use 'brew upgrade $@' instead.` I think is sufficient and more concise, thanks.